zerver/openapi/zulip.yaml

```diff
@@ -1925,13 +1848,13 @@ paths:
 
                         **Deprecated**.  We plan to remove the `pointer` as a concept in Zulip.
                       example: -1
-                    user_id:
-                      type: integer
-                      description: |
-                        The user's ID.
-                      example: 1
-                    profile_data:
-                      $ref: '#/components/schemas/profile_data'
+                    msg:
+                      type: string
+                    result:
+                      type: string
```

Comment:
Could we have the JsonSuccess component here as well to provide these keys?

Reply:
Oh, I see, it's a yamole bug.  OK.  yamole is software that we maintain (@YagoGG wrote it), so we should feel free to go into it and fix things directly.